pom.xml

```diff
@@ -98,6 +98,16 @@
                 <artifactId>farao-glsk-document-api</artifactId>
                 <version>${farao.core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.rte-france.farao</groupId>
+                <artifactId>rao-runner-api</artifactId>
+                <version>2.7.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.rte-france.farao</groupId>
+                <artifactId>rao-runner-spring-boot-starter</artifactId>
```

Comment:
You should use <artifactId>gridcapa-rao-runner-spring-boot-starter</artifactId> with version 
 <gridcapa.rao.runner.starter.version>0.1.1</gridcapa.rao.runner.starter.version>